tests/polytope_test.py

```diff
@@ -486,5 +486,14 @@ def is_glpk_present():
         return False
 
 
+def test_gurobipy_return_same_result_as_scipy():
```

Comment:
It is good to test the added functionality. However, the absence of the solver `gurobi` on Travis CI implies that this test fails. Skimming the licensing webpage of `gurobi` gives the impression that installing `gurobi` on Travis CI is probably far from its licensing procedure. So it seems that this test would need to not be invoked on Travis CI.

Reply:
Today I learned that Gurobi has a limited license that is included by default when installing `gurobipy`, so it is possible now to run tests with Gurobi in CI without requesting special permission[^1], assuming the model is not "too large" [^2].

[^1]: https://support.gurobi.com/hc/en-us/articles/360044290292-How-do-I-install-Gurobi-for-Python#h_01GRC369WKBMV6VFRCGJPRTSW5
[^2]: https://support.gurobi.com/hc/en-us/articles/360051597492-How-do-I-resolve-a-Model-too-large-for-size-limited-Gurobi-license-error